proposals/routing.md

```diff
@@ -0,0 +1,154 @@
+# Locale
+
+## Problem
+
+Currently for every URL we have a few different configurations.
+
+## Solution:
+
+We want to manage URLs in a consistent way, with only one configuration that can be used both for routing and for generating URLs.
+
+A centralised router object keep track of all the URL definitions. This router generates all the different routers (React Router, Express routing etc.) and URLs.
+
+### URL definitions
+
+Every URL needs a name and a pattern. We can use the React Router format for the pattern as it's the same as the one used by Express.
+
+```javascript
+var genericUrls = [
+    {
+        name: 'homepage',
+        pattern: '/',
+    },
+    {
+        name: 'about',
+        pattern: '/about/',
+    },
+];
```

Comment:
What about?
`var genericUrls = { homepage: { pattern: '/' }, name: { pattern: 'about' }}

Reply:
You can push this a bit further and have:
```
{
  homepage: '/',
  about: '/about/',
}
```

Reply:
Yeah I thought about it, but was not sure if it was a good or bad idea